modules/ROOT/nav.adoc

```diff
@@ -41,7 +41,7 @@
 * {cstar-data-migrator}
 ** xref:cdm-overview.adoc[]
 ** xref:cdm-steps.adoc[Migrate data]
-** xref:cdm-parameters.adoc[Parameters]
+//** xref:cdm-parameters.adoc[Parameters]
```

Comment:
Why is this being commented out instead of removed?

Reply:
Now completed remove that file along with other partial files too.